docs/docusaurus/docs/get_support.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_label: 'Get support'
+title: 'Get support'
+description: Learn what resources are avialable to help you resolve issues with GX Core.
+---
+
+GX Core is open source software (OSS) and relies on an active user community for its continued success. Input from users like you help GX and the community build its collective knowledge and improve GX Core. Knowing how and where to seek support will help you get the answers you need to resolve issues quickly.
```

Comment:
"Knowing how and where to seek support will help you get the answers you need to resolve issues quickly."

We can't guarantee users will be able to resolve their issues or even get an answer (helpful or not) via our community forums or support channels. Do you think we need a softer wording to that effect here? (FWIW I tried coming up with a rewrite, but couldn't figure out how to rephrase it well).

Reply:
@rachhouse I removed _quickly_. It's a start!

Reply:
👍